packages/commandkit/bin/common.mjs

```diff
@@ -72,16 +72,24 @@ const possibleFileNames = [
 export async function findCommandKitConfig(src) {
     const cwd = process.cwd();
     const locations = src ? [join(cwd, src)] : possibleFileNames.map((name) => join(cwd, name));
+    const foundConfigFiles = [];
 
     for (const location of locations) {
         try {
-            return await loadConfigInner(location);
+            const config = await loadConfigInner(location);
+            foundConfigFiles.push(location);
```

Comment:
why are we pushing config here? can't we just count the number?

Reply:
also I think this slows down the performance of bootstrapping process, do we really need this?

Reply:
> why are we pushing config here? can't we just count the number?

It's for actually logging the location of the file

Reply:
> also I think this slows down the performance of bootstrapping process, do we really need this?

It won't be that slow, it's for a rarer case anyway

Reply:
> > why are we pushing config here? can't we just count the number?
> 
> It's for actually logging the location of the file

I dont think that's logging the location, `loadConfigInner` loads config data

Reply:
> I dont think that's logging the location, `loadConfigInner` loads config data

It does log the location
<img width="779" alt="image" src="https://github.com/underctrl-io/commandkit/assets/140819896/f3ccfb62-59d3-49b6-89ac-c7553e7b4846">